### PR TITLE
internal: fix typos in comments

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Signed-off-by: yuxiaobo <yuxiaobogo@163.com>

### CODE REVIEW

**Comment by yuxiaobo96** [comment]
@dfawley 
Excuse me, can you help me add a label to this pr?

File: internal/channelz/types.go [hunk 1]
```diff
@@ -257,7 +257,7 @@ func (c *channel) deleteSelfFromTree() (deleted bool) {
 // channel, and its memory will be garbage collected.
 //
 // The trace reference count of the channel must be 0 in order to be deleted from the map. This is
-// specified in the channel tracing gRFC that as long as some other trace has reference to an entity,
+// specified in the channel tracing gRPC that as long as some other trace has reference to an entity,
```
Comment (dfawley) [inline internal/channelz/types.go]: gRFC is correct here; that's what our proposals are called.

Comment (yuxiaobo96) [reply internal/channelz/types.go]: Sorry, I will go to restore it.
Thank you.

**Comment by yuxiaobo96** [comment]
I have modified it.

**Comment by dfawley** [comment]
Thank you @yuxiaobo96!

**Pull request merged**